pages.id/common/alias.md

```diff
@@ -0,0 +1,28 @@
+# alias
+
+> Membuat alias -- kata-kata yang digantikan oleh utasan perintah (command)
+> Alias menjadi kadaluarsa sampai sesi shell saat ini berakhir, kecuali jika didefinisikan di file konfigurasi shell, misalnya `~/.bashrc`.
+
+- Menampilkan daftar semua alias:
+
+`alias`
+
+- Membuat alias generik:
+
+`alias {{word}}="{{command}}"`
```

Comment:
Please translate the arguments for get this easier for non-English speakers.

Reply:
fixed.
also added a few pages per suggestion.